hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/contract/precompile/TokenUpdatePrecompileSuite.java

```diff
@@ -402,13 +409,14 @@ final HapiSpec tokenUpdateSingleFieldCases() {
                                 .hasSupplyType(TokenSupplyType.INFINITE)
                                 .searchKeysGlobally()
                                 .hasAdminKey(MULTI_KEY)
-                                .hasPauseKey(MULTI_KEY)
```

Comment:
why are we removing the pause keys in these clauses?

Reply:
It is actually duplicated lines